token/services/ttx/auditor.go

```diff
@@ -146,11 +147,15 @@ func (a *AuditingViewInitiator) Call(context view.Context) (interface{}, error)
 	// Receive signature
 	logger.Debugf("Receiving signature for [%s]", a.tx.ID())
 	span.AddEvent("start_receiving")
-	signature, err := ReadMessage(session, time.Minute)
+	jsonSession := session2.NewFromSession(context, session)
+	signature, err := jsonSession.ReceiveRawWithTimeout(time.Minute)
 	if err != nil {
 		span.RecordError(err)
 		return nil, errors.WithMessage(err, "failed to read audit event")
 	}
+	if len(signature) == 0 {
```

Comment:
is this check needed? it might be misleading as to what is expected from the code if this can't happen. normally I expect that if err == nil, there data is valid.